Docs/source/usage/python.rst

```diff
@@ -252,11 +252,11 @@ An important object is ``Simulation.extension.warpx``, which is available during
 This object is the Python equivalent to the central ``WarpX`` simulation class and provides access to
 field ``MultiFab`` and ``ParticleContainer`` data.
 
-.. function:: pywarpx.picmi.Simulation.extension.warpx.getistep
+.. py:function:: pywarpx.picmi.Simulation.extension.warpx.getistep
 
-.. function:: pywarpx.picmi.Simulation.extension.warpx.gett_new
+.. py:function:: pywarpx.picmi.Simulation.extension.warpx.gett_new
 
-.. function:: pywarpx.picmi.Simulation.extension.warpx.evolve
+.. py:function:: pywarpx.picmi.Simulation.extension.warpx.evolve
```

Comment:
Should these have the ending `()` like the functions below? For example,
```
.. py:function:: pywarpx.picmi.Simulation.extension.warpx.getistep()
```
(and the next two below it).

Reply:
I think it might have worked like this in the past, but now added for consistency. Some of those might have (default) params that we may want to check.